manifests/base/jupyter-habana-notebook-imagestream.yaml

```diff
@@ -18,7 +18,7 @@ spec:
     # 1.13.0 Version of the image n
     - annotations:
         opendatahub.io/notebook-software: '[{"name":"Python","version":"v3.8"},{"name":"Habana","version":"1.13"}]'
-        opendatahub.io/notebook-python-dependencies: '[{"name":"Boto3","version":"1.34"},{"name":"Kafka-Python","version":"2.0"},{"name":"Matplotlib","version":"3.7"},{"name":"Numpy","version":"1.23"},{"name":"Pandas","version":"2.0"},{"name":"Scikit-learn","version":"1.3"},{"name":"Codeflare-SDK","version":"0.18"},{"name":"Scipy","version":"1.10"},{"name":"TensorFlow","version":"2.13"},{"name":"PyTorch","version":"2.1"},{"name":"ODH-Elyra","version":"3.16"}]'
+        opendatahub.io/notebook-python-dependencies: '[{"name":"Boto3","version":"1.34"},{"name":"Kafka-Python","version":"2.0"},{"name":"Matplotlib","version":"3.7"},{"name":"Numpy","version":"1.23"},{"name":"Pandas","version":"2.0"},{"name":"Scikit-learn","version":"1.3"},{"name":"Codeflare-SDK","version":"0.19"},{"name":"Scipy","version":"1.10"},{"name":"TensorFlow","version":"2.13"},{"name":"PyTorch","version":"2.1"},{"name":"ODH-Elyra","version":"3.16"}]'
```

Comment:
Does this version refer to [this one](https://github.com/opendatahub-io/notebooks/blob/main/habana/1.13.0/ubi8-python-3.8/Pipfile#L38)?

Reply:
hm, good catch Guilherme, will update tomorrow :slightly_smiling_face:

Reply:
Updated to 0.16 per the Pipfile and also checked [here](https://quay.io/repository/opendatahub/workbench-images/manifest/sha256:be95e83a76049c0a7eb4a89ce0df7359178fb2dd290ad50fcf2ad4e783e08185?tab=packages).

Thanks for this catch again, Guilherme!